packages/web3-validator/src/web3_validator.ts

```diff
@@ -0,0 +1,61 @@
+import Ajv from 'ajv';
+import { ethAbiToJsonSchema } from './utils';
+import { ValidationSchemaInput, Web3ValidationErrorObject, Web3ValidationOptions } from './types';
+import { ethKeyword } from './keywords/eth';
+import { Web3ValidatorError } from './errors';
+
+export class Web3Validator {
```

Comment:
This should be in web3 utils package.why we have new package : are there any **quantitative** constraint or objective we will achieve with it?

Reply:
After discussion we decided keep it as separate package.